cmd/build/helmify/main.go

```diff
@@ -104,6 +104,11 @@ func (ks *kindSet) Write() error {
 				obj = strings.Replace(obj, "      labels:", "      labels:\n{{- include \"gatekeeper.podLabels\" . }}", 1)
 			}
 
+			if kind == "ClusterRole" && name == "gatekeeper-manager-role" {
```

Comment:
If you update https://github.com/open-policy-agent/gatekeeper/blob/master/manifest_staging/charts/gatekeeper/templates/gatekeeper-manager-role-clusterrole.yaml I don't think you need this update in this file.